Snippets/Snippets_6/UnitTesting/AdditionalServices/MyMessage.cs

```diff
@@ -0,0 +1,6 @@
+namespace Snippets6.UnitTesting.AdditionalServices
+{
+    class MyMessage
+    {
+    }
+}
```

Comment:
well the still can be outside the region. but I think the additional files add more noise?